### PR TITLE
Inbox screens

### PR DESCRIPTION
**Pull request by shanewilsonpro**
## DESCRIPTION:
I changed the name from "Inbox" to "Chat" in the bottom tabs because I moved the activity page on home page. I completed three screens in chat; messages list, new message list, and conversation. The messages list on the screen that the user can view the messages. The messages only appear when a user or other sent the message to each other. New message list in screen that user can view their friends list, only appear from user's following. Conversation on the screen is between the user and another user to chat with. Thanks to the package I installed, it is able to text each other but I have not set up the function for firebase and store the messages. Lastly the activity list on the screen in home page, the user can view the activity in the list when another user likes their posts, comment or follow.

### Messages Screen:
Screen Shot 2021-07-17 at 1 51 26 PM
Screen Shot 2021-07-17 at 1 54 11 PM

### New Message Screen:
Screen Shot 2021-07-17 at 1 55 05 PM
Screen Shot 2021-07-17 at 1 55 30 PM

### Conversation Screen:
Screen Shot 2021-07-17 at 1 56 14 PM
Screen Shot 2021-07-17 at 1 57 00 PM

### Activity Screen:
Screen Shot 2021-07-17 at 2 00 34 PM
Screen Shot 2021-07-17 at 2 01 09 PM

## BONUS:
I have added the loading screen when logging in. I have updated the functionality of the authentications to be able the user to stay logged in when closing the app. I have updated the text component for all the screens. 

## PACKAGES INSTALLED AND DOCUMENTATION:
[Expo Video Thumbnails](https://docs.expo.io/versions/v41.0.0/sdk/video-thumbnails/)
[Gifted Chat](https://github.com/FaridSafi/react-native-gifted-chat)
[Raw Bottom Sheet](https://github.com/nysamnang/react-native-raw-bottom-sheet)
[Swipe List View](https://github.com/jemise111/react-native-swipe-list-view)

### CODE REVIEW

File: backlog.txt [hunk 1]
```diff
@@ -8,22 +8,7 @@ Sources might help:
...
-- https://www.youtube.com/watch?v=gvF6sFIPfsQ
-- https://github.com/ReactNativeSchool/getting-started-react-navigation-v5/blob/master/App/index.js
-- https://github.com/italoiz/react-navigation-reset-bottom-tabs-logout
-- https://reactnavigation.org/docs/auth-flow/
-- https://github.com/bithoven-dev/ReactNavigationAuthenticationFlowsWithHooks/blob/140f4b0b78/src/hooks/useAuth.js
-- https://github.com/itzpradip/react-navigation-v5-mix/blob/master/App.js
-- https://dev.to/embeddednature/create-an-authorization-flow-with-react-navigation-5-x-2pkh
-
-3RD ISSUE: Try to figure out on how to handle keyboard when focus on textinput in Post screen
+2ND ISSUE: Try to figure out on how to handle keyboard when focus on textinput in Post screen
```
Comment (cerberlink) [inline backlog.txt:11]: I think you can either put your docs within backlog via Trello or make a report that ticket on Issue.

File: src/components/typography/text.components.js [hunk 2]
```diff
@@ -25,14 +37,24 @@ const error = (theme) => `
...
-    font-size: ${theme.fontSizes.body};
+    font-size: ${theme.fontSizes.caption};
+`;
+
+const numbers = (theme) => `
+    font-family: ${theme.fonts.body_600};
+    font-size: ${theme.fontSizes.caption};
 `;
 
 const date = (theme) => `
```
Comment (cerberlink) [inline src/components/typography/text.components.js:60]: What happened with this? Is that accidentally deleting?

File: src/features/account/screens/login.screen.js [hunk 3]
```diff
@@ -49,10 +35,10 @@ const OrSection = styled.View`
 `;
```
Comment (cerberlink) [inline src/features/account/screens/login.screen.js:35]: this code and on line 68 seem not used. If you think this is not used, you probably can remove it. We don't want to see unused code floating in this codebase.

**Review by cerberlink — APPROVED** [review]
LGTM! Great job!

File: backlog.txt (hunk 1 above)
Comment (AustinVaday) [reply backlog.txt:11]: +1 to report issues using Github instead of keeping them in a text file here. 

https://github.com/AustinVaday/SFH-App/issues

**Review by AustinVaday — APPROVED** [review]
This is awesome, thanks for doing this change!

**Pull request merged**